config/initializers/mail.rb

```diff
@@ -0,0 +1,11 @@
+ActionMailer::Base.smtp_settings = {
+	address: "smtp.mandrillapp.com",
+  port: 587,
+  enable_starttls_auto: true,
+  user_name: ENV['MANDRILL_USERNAME'],
+  password: ENV['MANDRILL_PASSWORD'],
+  authentication: "login"
+}
+
+ActionMailer::Base.delivery_method = :smtp
+ActionMailer::Base.default charset: "utf-8"
```

Comment:
We should move this code to config/environments/production.rb to make sure we actually send emails using mandrill only in production and we can use a gem like letter_opener to handle emails in development. What do you think @sebasoga and @linatodi ?

Configuring letter opener is pretty simple:

Add the gem to the Gemfile.
group :development do
  gem 'letter_opener'
end
Configure action mailer to use letter_opener in development.
# config/environments/development.rb

config.action_mailer.delivery_method = :letter_opener